src/main/java/com/breskul/bibernate/util/Triple.java

```diff
@@ -0,0 +1,9 @@
+package com.breskul.bibernate.util;
+
+public record Triple<F, S, T>(F first, S second, T third) {
```

Comment:
For readability, I advise using more specific wrapper classes. `Triple` is really hard to read and understand what is going on.

Reply:
done